### PR TITLE
Create destroy

### PR DESCRIPTION
**Pull request by latter-jacob**

### CODE REVIEW

File: app/controllers/programs_controller.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def create
   end
 
   def new
-    if current_user.admin?
+    if current_user.try(:admin?)
```
Comment (mlg-) [inline app/controllers/programs_controller.rb]: `before_action`


`before_action`


`before_action`

etc

- a poem by mlg

File: spec/features/admin_deletes_program_spec.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+feature 'admin deletes a program', %{
+  As an admin
+  I want to be able to delete a program
+} do
+
+  # [ ] Admins are able to verify their credentials
```
Comment (mlg-) [inline spec/features/admin_deletes_program_spec.rb]: Split the admin can log in tests into their own spec. It's not really related to the program deletion.

File: spec/features/admin_deletes_program_spec.rb [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+  scenario 'admin deletes program successfully' do
+    admin = FactoryGirl.create(:user, admin: true)
+    program = FactoryGirl.create(:program)
+
+    visit new_user_session_path
+    fill_in 'Email', with: admin.email
+    fill_in 'Password', with: admin.password
+    click_button 'Log in'
+    expect(page).to have_content('Signed in successfully')
```
Comment (mlg-) [inline spec/features/admin_deletes_program_spec.rb]: You could abstract the process of logging in out into a spec helper module.

File: app/controllers/programs_controller.rb (hunk 1 above)
Comment (radavis) [inline app/controllers/programs_controller.rb]: Please read through the lesson on [Devise authorization](https://learn.launchacademy.com/lessons/devise-authorization) for examples of how to limit access to particular controller actions in a DRYer way.

**Pull request merged**